reporter/src/funTest/kotlin/TestData.kt

```diff
@@ -452,8 +452,8 @@ val ORT_RESULT = OrtResult(
 )
 
 val VULNERABILITY = Vulnerability(
-    "CVE-2021-1234",
-    listOf(
+    id = "CVE-2021-1234",
```

Comment:
Would it make sense to add the new attributes to the test data?

Reply:
This test data is only used for testing the evaluator currently. 
I wouldn't want to add a test to the evaluator which covers `summary` or `description` as I believe these attributes are irrelevant for the rules, e.g. they are free text. Without adding a test case to the evaluator I believe it doesn't make sense to add to the test data. Are you ok with leaving it as-is?

Reply:
Ok.